README.md

```diff
@@ -265,7 +267,15 @@ The following targets are currently supported:
 ```json
```

Comment:
Change to `js` so that comments won't appear as errors

Reply:
✔️